opentelemetry-semantic-conventions/src/resource.rs

```diff
@@ -21,7 +21,6 @@
 //! let _tracer = TracerProvider::builder()
 //!     .with_config(config().with_resource(Resource::new(vec![
 //!         KeyValue::new(semconv::resource::SERVICE_NAME, "my-service"),
-//!         KeyValue::new(semconv::resource::SERVICE_NAMESPACE, "my-namespace"),
```

Comment:
this was auto-generated after templates were changed, right?

Reply:
Any discrepancy in the committed and auto-generated files would have been caught in CI. So should be good.

Reply:
Are you sure about that? The `init()` to `build()` change was a correction of a missed change in the `metrics` template, and that had not been caught by CI.

Reply:
Oh sorry, got confused with the proto generated file comparison, the semconv files discrepancy is not caught. @bantonsson  - Can you regenerate the files in new PR.

Reply:
I did regenerate the files before submitting this PR. That’s how I caught the mismatch in the `metrics` files.

Reply:
Got it, yes directly modifying the `metrics.rs` was unfortunate and should have been caught in the CI (the way we do it for proto).